config/samples/dataplane_v1beta1_openstackdataplanenode_edpm_compute_0_full.yaml

```diff
@@ -7,6 +7,12 @@ spec:
   ansibleHost: 192.168.122.100
   ansibleSSHPrivateKeySecret: dataplane-ansible-ssh-private-key-secret
   node:
+    services:
+      - configure-network
+      - validate-network
```

Comment:
this si a nit or question more then a request for change but should we be validating before or after?

is there a general pattern or nameing convention we should set here going forward

i.e. is validate-network ensuring the input is correct or is it ensuring the network is not configured as expected.

i would assume the former in which case i think the order is reversed.

configure-network should fail if it was not able to configure the network requested so if its the latter (validating that configure-network did the right thing) then i think we should combine the two

Reply:
configure-network applies the user requested network configuration. validate-network then validates it. configuring something does not imply validation, just in the same way that if you set an option in a conf file, it is not validated until later when used (in most cases anyway).